docker/plugins.txt

```diff
@@ -61,21 +64,24 @@ pipeline-stage-view:2.19
 plain-credentials:1.7
 plugin-util-api:2.4.0
 popper-api:1.16.1-2
+popper2-api:2.9.3-1
 resource-disposer:0.16
 scm-api:2.6.5
 script-security:1.78
 slack:2.48
 snakeyaml-api:1.29.1
 ssh-agent:1.23
 ssh-credentials:1.19
+sshd:3.1.0
 structs:1.23
 timestamper:1.13
 token-macro:266.v44a80cf277fd
 trilead-api:1.0.13
 variant:1.4
+windows-slaves:1.8
```

Comment:
Is this one a dependency of something else? I don't think we're running any Windows builds, are we?

Reply:
As per comment above

Reply:
https://jenkins.tdr-management.nationalarchives.gov.uk/pluginManager/installed says that the HSTS plugin might be depending on it, which is odd because all HSTS does is add an extra security header to Jenkins server responses.

Might be worth removing it from the list and seeing whether Jenkins seems to work?